auslib/test/admin/views/test_rules.py

```diff
@@ -38,6 +38,25 @@ def testNewRulePostJSON(self):
         self.assertEquals(r[0]['priority'], 33)
         self.assertEquals(r[0]['data_version'], 1)
 
+    def testNewRuleWithProductAdminPermission(self):
+        data = json.dumps(dict(
+            backgroundRate=31, mapping="a", priority=33, product="Firefox",
+            update_type="minor", channel="nightly"
+        ))
+        ret = self._post("/rules", data=data, headers={"Content-Type":
+                                                       "application/json"},
+                         username="billy")
```

Comment:
You shouldn't need to set Content-Type here nor dump to json, those should both be handled by ViewTest._post.